release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,90 @@
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: addtl-release-notes
+
+Release notes for additional related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="4,7a",options="header"]
+|===
+|Component or product |Release note documentation
+
+|AWS Load Balancer (ALB) Operator
+|xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer Operator release notes]
+
+|Compliance Operator
+|xref:../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance Operator release notes]
+
+|{rh-dev-hub} Operator
+|link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/index[Release notes]
+
+|External DNS Operator
+|xref:../networking/external_dns_operator/external-dns-operator-release-notes.adoc#external-dns-operator-release-notes[External DNS Operator release notes]
+
+|File Integrity Operator
+|xref:../security/file_integrity_operator/file-integrity-operator-release-notes.adoc#file-integrity-operator-release-notes-v0[File Integrity Operator release notes]
+
+|{descheduler-operator}
+|xref:../nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc#nodes-descheduler-release-notes[{descheduler-operator} release notes]
+
+|{logging-uc}
+|xref:../observability/logging/logging_release_notes/logging-5-9-release-notes.adoc#logging-5-9-release-notes[{logging-uc} -> Release notes]
+
+|Migration Toolkit for Containers (MTC)
+|xref:../migration_toolkit_for_containers/release_notes/mtc-release-notes.adoc#mtc-release-notes[Migration Toolkit for Containers release notes]
+
+|Network Observability Operator
+|xref:../observability/network_observability/network-observability-operator-release-notes.adoc#network-observability-operator-release-notes[Network Observability Operator release notes]
+
+|{openshift-dev-spaces-productname}
+|link:https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/3.16#Release%20Notes[Release Notes]
+
+|{DTProductName}
+|xref:../observability/distr_tracing/distr-tracing-rn.adoc#distr-tracing-rn[Release notes for the {DTProductName}] (Tempo and Jaeger)
+
+|{gitops-title}
+|link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.13/html/release_notes/index[Release notes]
+
+|{openshift-local-productname}
+|link:https://docs.redhat.com/en/documentation/red_hat_openshift_local/2.41/html/release_notes_and_known_issues/index[Release notes and known issues]
```

Comment:
Are we going to update the link for each release of OpenShift Local? Since we do the release in every 3 weeks?

Reply:
Honestly I think we're probably better off just linking to https://docs.redhat.com/en/documentation/red_hat_openshift_local/ which redirects to the latest version of the splash page. And then the release notes are easily spotted there. Unless a writer from the OpenShift Local wants to own updating the deep link each release.

Reply:
@adellape I think https://docs.redhat.com/en/documentation/red_hat_openshift_local is what I wanted to have it here instead deep link.